Doc/library/contextlib.rst

```diff
@@ -529,7 +529,8 @@ Functions and classes provided:
 
       Enters a new context manager and adds its :meth:`__exit__` method to
       the callback stack. The return value is the result of the context
-      manager's own :meth:`__enter__` method.
+      manager's own :meth:`__enter__` method.The :meth:`__enter__` method 
+      returns the ExitStack instance, and performs no additional operations.
```

Comment:
Can we make a link for ExitStack?

Reply:
Sure :).

Reply:
Thanks for your comments. It is updated in the latest commit.